xla/service/gpu/llvm_gpu_backend/gpu_backend_lib.cc

```diff
@@ -708,6 +711,9 @@ absl::StatusOr<std::string> CompileToPtx(
   return ptx;
 }
 
+int PtxVersionFromCudaVersionForTest(se::ToolVersion tool_version) {
+  return DetermineHighestSupportedPtxVersionFromCudaVersion(tool_version);
+}
```

Comment:
Why not expose the original function?

Reply:
Exposed the original function in the `nvptx` namespace.